test/exercism/homework_test.rb

```diff
@@ -0,0 +1,72 @@
+require_relative '../test_helper'
+require_relative '../active_record_helper'
+require_relative '../../lib/exercism'
+require_relative '../../lib/exercism/homework'
+
+class HomeworkTest < Minitest::Test
+  include DBCleaner
+
+  def alice
+    @alice ||= User.create(username: 'alice')
+  end
+
+  def teardown
+    super
+    @alice = nil
+    @language = nil
+    @skipped = nil
+    @fetched = nil
+    @recent = nil
+  end
```

Comment:
I actually prefer to not have so many helpers in setup/teardown (and not using instance variables), and rather creating the data within the test itself.